packages/salesforcedx-vscode-core/package.json

```diff
@@ -407,11 +407,6 @@
           "type": ["boolean"],
           "default": true,
           "description": "%show_cli_success_msg_description%"
-        },
-        "salesforcedx-vscode-core.isv-debugger.enabled": {
```

Comment:
Did you check with @GaryFNC on how he did it? The reason I ask is that if you remove this, it might mean that there is no longer a default, which would mean https://github.com/forcedotcom/salesforcedx-vscode/blob/59698fed560bb6fb70c3c86f5ee9daca2cb8fc34/packages/salesforcedx-vscode-core/package.json#L260 could get into a weird situation since the value is null (unless it does the JavaScript truthy checking).

Could you check what happens if you don't have the setting set and then you try to invoke the command? Any errors?

Reply:
The command disappears.